src/kOS/Binding/BindingsUniverse.cs

```diff
@@ -152,6 +152,8 @@ public override void AddTo(SharedObjects shared)
             }
 
             shared.BindingMgr.AddGetter("VERSION", () => Core.VersionInfo);
+
+            shared.BindingMgr.AddGetter("STEERINGMANAGER", () => SteeringManager.GetInstance(shared));
```

Comment:
No good reason, to be honest.  It makes a lot of sense to put it in flightstats.  I just hadn't honestly considered it.